profiles/hfcl_ion4.yml

```diff
@@ -0,0 +1,7 @@
+profile: hfcl_ion4
+target: ipq40xx
+subtarget: generic
+description: Build image for the HFCL ION4
+image: bin/targets/ipq40xx/generic/openwrt-ipq40xx-generic-hfcl_ion4-squashfs-nand-sysupgrade.bin
+packages:
+    - uboot-envtools
```

Comment:
this should got into DEFAULT_PACKAGES in the image/Makefile

Reply:
@blogic sure we will move this to image/Makefile.
Please let us know if futher any changes required.